documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

```diff
@@ -140,6 +140,7 @@ Depending on how you apply OAuth 2.0 authentication, and the type of authorizati
 <7> Set this to `false` to disable the OAUTHBEARER mechanism on the listener. At least one of PLAIN or OAUTHBEARER has to be enabled. Default is `true`.
 <8> Set to `true` to enable PLAIN authentication on the listener, which is supported for clients on all platforms.
 <9> Additional configuration for the PLAIN mechanism. If specified, clients can authenticate over PLAIN by passing an access token as the `password` using an `$accessToken:` prefix.
+For production, always use HTTPs.
```

Comment:
To me this reads as 'more than one HTTP' rather than 'https://' which I guess was intended.
```suggestion
For production, always use `https://` urls.
```

Reply:
Agree with Marko that the `HTTPs` is confusing. Marko's suggestion works for me. You could also just say something like _For production, always use encryption._ or something I guess.

Reply:
Typo. Which was reproduced from a typo elsewhere. I've updated in both places as suggested, which is clearer.